testsuite/makefiles/summarize.awk

```diff
@@ -41,6 +41,14 @@ function record_skip() {
     clear();
 }
 
+function record_na() {
+    check();
+    if (!(key in RESULTS)) ++nresults;
+    RESULTS[key] = "n";
+    if (curdir in SKIPPED) SKIPPED[curdir] = 1;
```

Comment:
I'm not familiar with the way SKIPPED work; why are you setting `SKIPPED[curdir]` as in `record_skip` here, instead of deleting it as in `record_pass`? Would there a be a difference?

Reply:
An `n/a` test is still a kind of skipped test. This table is just used for detecting whether an entire directory has been skipped.